parser-utils/Cargo.toml

```diff
@@ -5,3 +5,9 @@ authors = ["Victor Porof <vporof@mozilla.com>", "Richard Newman <rnewman@mozilla
 
 [dependencies]
 combine = "2.1.1"
+
+[dependencies.edn]
+  path = "../edn"
+
+[dependencies.mentat_query]
```

Comment:
Can you explain why parsing utilities would depend on the structure of queries?  All the parsing utilities are likely to be abstract over their result type.